tests/integ/modin/test_to_numpy.py

```diff
@@ -58,8 +58,9 @@ def test_to_numpy_basic(data, pandas_obj, func):
             assert_array_equal(df.values, native_df.values)
     if pandas_obj == "Series":
         with SqlCounter(query_count=1):
-            res = df.to_list()
-        expected_res = native_df.to_list()
+            res = df.to_numpy()
+        expected_res = native_df.to_numpy()
```

Comment:
even thought this test is wired that we have to_list inside to_numpy test, do we have test for to_list? that seems a supported API

Reply:
Then i don't think you need to change this test now, can you revert the changes here for now, add a ticket to refactor that test and add some tests for to_list

Reply:
reverted and filed SNOW-1652384